README.md

```diff
@@ -64,6 +64,28 @@ $config->addAppInstance('local')
 return $config;
 ````
 
+## CLI Examples
+
+````bash
+phpsu sync production --dry-run
+phpsu sync p --no-db
+phpsu sync p --no-fs
+phpsu sync production testing --all
+````
+
+### PHP API Examples
```

Comment:
this we should leave out for now and put more though on it in the next milestone. Otherwise i'd approve it